tests/agent/__init__.py

```diff
@@ -0,0 +1,48 @@
+from contextlib import contextmanager
+import requests
+import time
+from urlparse import urljoin
+
+from tests.fixtures import default
+
+
+@contextmanager
+def remote_config(url, sampling_rate=1.0):
```

Comment:
since `url` is used a lot I think it should be more descriptive, how about `kibana_url` ?